src/sairajJDec19/assignment3/Question10.java

```diff
@@ -0,0 +1,22 @@
+package sairajJDec19.assignment3;
+
+public class Question10 {
+	
+	int getSum(int[]a, int[]b) {
+		int arraySum=0;
+		for(int i=0; i<a.length; i++) {
+			for(int j =0; j<b.length; j++)
+				if(a[i]==b[j]) {
```

Comment:
Hope You are running and check your code.
what is significant if **if** condition.